### ISSUE TITLE
A11y: Buttons present inside editor section are not visible in High Contrast Black mode.

### ISSUE THREAD

**Issue by lumills** [ckeditor/ckeditor4#4379]
Any environment

When using high contrast black mode, some buttons are not visible due to Opacity property of ckeditor button class. (Currently 0.3 for disabled buttons)

![image](https://user-images.githubusercontent.com/58861998/99438142-3e604980-28c8-11eb-9ba6-3328563bac80.png)

​OS Details:​
Windows Server 2019 Version 1809(OS Build 17763.914)
​
Browser Details:​
Chromium Edge Version 81.0.416.58 (Official build) beta (64-bit)

​Repro Steps:​​
1.Turn on High Contrast and open any ckeditor instance, demo site would suffice.

7.Verify user is able to view the buttons inside editor section or not.
​​
Actual Result:​​
Buttons present inside editor section are not visible in High Contrast Black mode.
Expected Result:​​
Buttons present inside editor section should be visible in High Contrast Black mode.
User Impact:​​
Low vision users can not see all controls if few controls are not visible in high contrast mode.
​
MAS Reference:​​
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B0BF582E1-6800-4E43-8947-DA015AA28B2B%7D&file=MAS%201.4.3%20%E2%80%93%20Contrast%20(Minimum).docx&action=default&mobileredirect=true

**Comment by f1ames** [comment]
I can confirm that disabled buttons in High Contrast Mode are hardly visible:

![](https://user-images.githubusercontent.com/1061942/99512429-bb072c80-2989-11eb-8d39-3aefde3544e0.png)

**Comment by f1ames** [comment]
It looks like this issue is caused by incorrect HC mode detection in Chromium Edge: https://github.com/ckeditor/ckeditor4/blob/71e0962bab676ec96b54d7795ad1025c4cdd271b/core/_bootstrap.js#L11-L13

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/4413

**Comment by sryarama** [comment]
A quick question, when will you release 4.15.2?

**Comment by f1ames** [comment]
Hi @sryarama, the next release containing this fix is planned for the end of January / beginning of February, so most probably within a month.
